requirements/extras/brotli.txt

```diff
@@ -1,2 +1,2 @@
 brotlipy>=0.7.0;platform_python_implementation=="PyPy"
-brotli>=1.0.0;platform_python_implementation=="CPython"
+brotli==1.0.7;platform_python_implementation=="CPython"
```

Comment:
This should not be changed at all.

Reply:
I wasn't able to build locally using tox without this, if I remember correctly.

Reply:
I'll revert this as well in anticipation of another test failing.